zds/notification/models.py

```diff
@@ -206,15 +207,20 @@ def mark_notification_read(self, content):
             raise Exception('Object content of notification must be defined')
 
         content_notification_type = ContentType.objects.get_for_model(content)
-        try:
-            notification = Notification.objects.get(subscription=self,
-                                                    content_type__pk=content_notification_type.pk,
-                                                    object_id=content.pk, is_read=False)
-            if notification is not None:
-                notification.is_read = True
-                notification.save()
-        except Notification.DoesNotExist:
-            pass
+        notifications = list(Notification.objects.filter(subscription=self,
+                                                         content_type__pk=content_notification_type.pk,
+                                                         object_id=content.pk, is_read=False))
+        # for some reason, it appears that some notification can be subscribed twice.
```

Comment:
Je propose donc : 

`# handles cases where a same subscription lead to several notifications`